packages/e2e/src/factories.ts

```diff
@@ -322,7 +322,8 @@ const patchInitializeTxToRespectEpochBoundary = <T extends ObservableWallet>(
  * @returns an object containing the wallet and providers passed to it
  */
 export const getWallet = async (props: GetWalletProps) => {
-  const { env, idx, logger, name, polling, stores, customKeyParams, keyAgent, witnesser } = props;
+  const { env, idx, logger, name, polling, stores, customKeyParams, witnesser } = props;
+  const keyAgent = props.keyAgent;
```

Comment:
Why this change?

And moreover, how could it be that the eslint rule `unicorn/consistent-destructuring` is not applied here?

Reply:
Great catch!
Weird indeed. The change was due to me experimenting, and forgot to roll it back

Reply:
Just out of curiosity, mainly about the eslint rule...
I think no actions are required here